app-misc/brewtarget/brewtarget-4.0.15.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# All tests currently timeout even with a large timeout multiliplier
+# so they are commented out and disabled for now
+
+EAPI=8
+
+PLOCALES="ast ca cs da de el en en_GB es et eu fr gl hr hu it ja lv nb nl pl pt pt_BR pt_PT ru sr sv tr zh"
+#inherit meson plocale xdg-utils virtualx
+inherit meson plocale xdg-utils
```

Comment:
Please drop plocales, we do not obsess about small files any longer.

Cleanup the commented out line also.

Reply:
Done